.github/workflows/ci.yaml

```diff
@@ -53,9 +138,9 @@ jobs:
       - name: Install python dependencies
         run: |
           make setup
-      - name: Configure environment
+      - name: Configure neuro
         run: |
-          neuro config login-with-token ${{ secrets.NEURO_TOKEN }} ${{ secrets.NEURO_STAGING_URL }}
+          neuro config login-with-token ${{ env.NEURO_TOKEN }} ${{ env.NEURO_STAGING_URL }}
```

Comment:
Minor: I see no difference between `env` and `secrets` contexts here; as I see you never use `NEURO_TOKEN` as *environment variable* but substitute the text value into executed command explicitly.

Reply:
As you wish, though